README.md

```diff
@@ -0,0 +1,189 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#Hello-microverse)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  -- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello-microverse] <a name="Hello-microverse"></a>
+
+**[Hello-microverse]** this was mini-projects that upkill to create a feature branch and test linters in order to have professional-looking code.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <summary>Javascript runtime environment</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node JS</a></li>
+  </ul>
```

Comment:
- [ ] I think it would be better if you update the above information with the correct data, as you used `HTML` and `CSS` to  build your project, there are the one which should be listed there:
```
  <ul>
    <li><a href="#">HTML</a></li>
    <li><a href="#">CSS</a></li>
  </ul>
```